commit 535dfe4e3630f8dbbbcdad79776f79e87591fa85
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Mon Oct 8 18:26:15 2018 +0100

    staging: bcm2835-camera: Correct V4L2_CID_COLORFX_CBCR behaviour
    
    With V4L2_CID_COLORFX_CBCR calling ctrl_set_colfx it was incorrectly
    assigning the colour values to the enable field of dev->colourfx
    instead of the u and v fields.
    
    Correct the assignments.
    
    Reported as a Coverity issue
    Detected by CoverityScan CID#1419711 ("Unused value")
    
    Reported-by: Colin Ian King <colin.king@canonical.com>
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/staging/vc04_services/bcm2835-camera/controls.c b/drivers/staging/vc04_services/bcm2835-camera/controls.c
index 21409f3729a4..c4c1c191bce8 100644
--- a/drivers/staging/vc04_services/bcm2835-camera/controls.c
+++ b/drivers/staging/vc04_services/bcm2835-camera/controls.c
@@ -578,8 +578,8 @@ static int ctrl_set_colfx(struct bm2835_mmal_dev *dev,
 
 	control = &dev->component[COMP_CAMERA]->control;
 
-	dev->colourfx.enable = (ctrl->val & 0xff00) >> 8;
-	dev->colourfx.enable = ctrl->val & 0xff;
+	dev->colourfx.u = (ctrl->val & 0xff00) >> 8;
+	dev->colourfx.v = ctrl->val & 0xff;
 
 	ret = vchiq_mmal_port_parameter_set(dev->instance, control,
 					    MMAL_PARAMETER_COLOUR_EFFECT,
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1266_535dfe4e3630f8dbbbcdad79776f79e87591fa85.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1266_535dfe4e3630f8dbbbcdad79776f79e87591fa85.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
